library/core/src/ptr/const_ptr.rs

```diff
@@ -483,8 +483,9 @@ impl<T: ?Sized> *const T {
     ///
     /// This operation itself is always safe, but using the resulting pointer is not.
     ///
-    /// The resulting pointer "remembers" the [allocated object] that `self` points to; it must not
-    /// be used to read or write other allocated objects.
+    /// The resulting pointer "remembers" the [allocated object] that `self` points to
+    /// (This is called "[Provenance](ptr/index.html#provenance)").
```

Comment:
```suggestion
    /// (this is called "[Provenance](ptr/index.html#provenance)").
```
same below. Then please squash the commits.